drivers/hmis/app/graphql/types/hmis_schema/query_type.rb

```diff
@@ -416,6 +416,26 @@ def form_definitions(filters:)
       scope.order(updated_at: :desc)
     end
 
+    field :form_identifier, Types::Forms::FormIdentifier, null: true do
+      argument :identifier, String, required: true
+    end
+    def form_identifier(identifier:)
+      raise 'Access denied' unless current_user.can_configure_data_collection?
+
+      Hmis::Form::Definition.where(identifier: identifier).published.first
```

Comment:
perhaps order? 
```suggestion
      Hmis::Form::Definition.where(identifier: identifier).order(version: :desc).published.first
```

Or maybe it should use the same scope as the the `form_identifiers` resolver
```suggestion
      Hmis::Form::Definition.non_static.latest_versions.where(identifier: identifier).first
```

Reply:
Good idea, I think I like using the same scope as the `form_identifiers` resolver. In theory they should all resolve to similar objects anyway with the same drafts/published version/etc, but it's good to make it more predictable and consistent